install.sh error on Ubuntu 15.04 (Vivid)

```
[azk] Checking platform.......................................[ DONE ]
[azk]   Detected platform: linux, x64
[azk]   Detected distribution: ubuntu, 15.04
[azk] Enabling sudo...........................................[ WAIT ]
[azk]   echo sudo enabled
[azk] Enabling sudo...........................................[ DONE ]

[azk] [ERROR]   Unsupported Ubuntu version.
[azk] [ERROR]   Feel free to ask support for it by opening an issue at:
[azk] [ERROR]     https://github.com/azukiapp/azk/issues
[azk] [ERROR] Failed to install azk.
```

According to Requirements:
Distributions (tested): Ubuntu 12.04/14.04/15.04 and Fedora 20/21/22

--- a/docs/content/en/installation/linux.md
+++ b/docs/content/en/installation/linux.md
@@ -4,7 +4,7 @@
 
 ## Requirements
 
-* **Distributions (tested)**: Ubuntu 12.04/14.04/15.04 and Fedora 20/21/22
+* **Distributions (tested)**: Ubuntu 12.04/14.04/15.10 and Fedora 20/21/22
 * **Architecture**: 64-bits
 * [Docker][docker] 1.8.1
 * Not running any services on ports `80` and `53`

--- a/docs/content/pt-BR/installation/linux.md
+++ b/docs/content/pt-BR/installation/linux.md
@@ -4,7 +4,7 @@
 
 ## Requisitos
 
-* **Distribuições (testadas)**: Ubuntu 12.04/14.04/15.04 e Fedora 20/21/22
+* **Distribuições (testadas)**: Ubuntu 12.04/14.04/15.10 e Fedora 20/21/22
 * **Arquitetura**: 64-bits
 * [Docker][docker] 1.8.1
 * Não estar rodando nenhum serviço nas portas `80` e `53`

